crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -703,10 +723,31 @@ impl PlotItem for Text {
             self.color
         };
 
-        let galley =
-            self.text
-                .clone()
-                .into_galley(ui, Some(false), f32::INFINITY, TextStyle::Small);
+        let font = if self.font_size.is_some() {
+            let name_font = format!("Plot_{}_{}", self.font.clone(), self.font_size.unwrap());
+            if ui
+                .style()
+                .text_styles
+                .get_key_value(&TextStyle::Name(name_font.clone().into()))
+                .is_none()
+            {
+                ui.style_mut().text_styles.insert(
+                    TextStyle::Name(name_font.clone().into()),
+                    FontId {
+                        size: self.font_size.unwrap(),
+                        family: Default::default(),
+                    },
+                );
+            }
+            TextStyle::Name(name_font.into())
```

Comment:
this is a bad idea. There is no need to modify the ui style. Just `FontId` instead!

Reply:
Thank !  I need to integrate these concepts particularly FontId and AB_glyph. Especially since what I really wanted was for the text to appear frozen in the plot, ie when zooming in the text should appear to grow.